### ISSUE TITLE
Cannot run agartest or the hello.c example on Fedora Linux

### ISSUE THREAD

**Issue by bkeys** [JulNadeauCA/libagar#75]
I follow the tutorials here
https://libagar.org/docs/inst/linux.html
and when I build the example it mentions it gives me this error:
```
./hello: error while loading shared libraries: libag_gui.so.8: cannot open shared object file: No such file or directory
```
And I am unable to run agartest because I get a similar error saying it's not loading libag_math.so
I am on Fedora Linux 38 right now.
As a side note, would you be open to a CMake based build system?

**Comment by JulNadeauCA** [comment]
Can you check that `/usr/local/lib` exists in your `/etc/ld.so.conf` file?
Then, run the command `ldconfig` to refresh your dynamic linker cache.
For some reason, some Linux distributions need this.

Yes, it would be nice to offer some CMake build scripts (those could even be generated by BSDBuild), but I haven't had the time to look into it yet.

**Comment by JulNadeauCA** [comment]
This appears to be solved. Please feel free to reopen the issue if I missed anything.
Regarding CMake, it is now available in the development sources and will be included in the 1.7.1 release.
